workload/run.yml

```diff
@@ -0,0 +1,20 @@
+---
+# this is a sample of workload
+# prerequisite: 
+# - we rely on the rally template mechanism
+# 
+rally: 
+    enabled: true
```

Comment:
What is the purpose of the `enable` key at the top level? 

Maybe to prepare the field for other stress libraries, But, my philosophy is "If you don't use it, remove it".